manifests/local-storage.yaml

```diff
@@ -82,6 +82,26 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        livenessProbe:
+          exec:
+            command:
+              - /bin/sh
+              - -ec
+              - |
+                pgrep local-path-provisioner
```

Comment:
I don't see that this adds a lot of value. `local-path-provisioner` is the command for the only container in the pod; if that command exits then the container will exit and be restarted with or without the added health check. Adding a health check to make sure it is still running, when its exit will cause the container to exit, doesn't seem like a meaningful addition.

Reply:
I agree that this check is not very useful. Do you know of a possible check that has more value?

Reply:
It would be nice if the local-path-provisioner had a health check listener, but given that it doesn't, I don't think there is a meaningful health check to run at the moment.

Reply:
Alright, I've removed it.